tasks/main.yml

```diff
@@ -1,11 +1,51 @@
 # SPDX-License-Identifier: MIT
 ---
-# Put the tasks for your role here.
+- name: Assume managing timesync if timesource is set
+  set_fact:
+    ad_integration_manage_timesync: true
+  when:
+    - ad_integration_timesync_source is not none
```

Comment:
It is a very minor thing, but when `ad_integration_timesync_source == ""`, it is `not none`. I guess you don't want to set `ad_integration_manage_timesync` to `true`.

Reply:
Is it correct that it should be changed to the following?
~~~
when:
  - ad_integration_timesync_source is not none
  - ad_integration_timesync_source != ""
~~~

Reply:
Yes, I think so. Thanks!

Reply:
Fixed.

Reply:
Sorry, ansible-lint does not like it. (;_;)
```
# .ansible-lint
warn_list:  # or 'skip_list' to silence them completely
  - empty-string-compare  # Don't compare to empty string
```
You'd better do something like this...
```
- ad_integration_timesync_source | length > 0
- ```